package.json

```diff
@@ -15,7 +15,10 @@
     "body-parser": "^1.18.3",
     "dotenv": "^10.0.0",
     "express": "^4.16.4",
+    "express-mysql-session": "^2.1.7",
+    "express-session": "^1.17.2",
     "mysql2": "^2.3.2",
+    "package-lock-only": "0.0.4",
     "passport": "^0.5.0",
     "passport-local": "^1.0.0",
     "sass": "^1.43.4",
```

Comment:
I must have done this to you when I had you run the command once, lol

It's `npm install --package-lock-only`
we must have done `npm install package-lock-only`

we need to have the `--` in there